lib/nostrum/voice.ex

```diff
@@ -87,15 +87,38 @@ defmodule Nostrum.Voice do
       - `:url` Input will be [any url that `ffmpeg` can read](https://www.ffmpeg.org/ffmpeg-protocols.html).
       - `:pipe` Input will be data that is piped to stdin of `ffmpeg`.
       - `:ytdl` Input will be url for `youtube-dl`, which gets automatically piped to `ffmpeg`.
+    - `options` - See options section below.
+
 
   Returns `{:error, reason}` if unable to play or a sound is playing, else `:ok`.
 
+  ## Options
+    - `:start_pos` (string) - The start position of the audio to be played. Defaults to beginning.
+    - `:duration` (string) - The duration to of the audio to be played . Defaults to entire duration.
+    - `:realtime` (boolean) - Make ffmpeg process the input in realtime instead of as fast as possible. Defaults to true.
+    - `:volume` (number) - The output volume of the audio. Default volume is 1.0.
```

Comment:
Do you think there's any value in normalizing this number to be more equivalent to how most people think about volume? I'm not too familiar with ffmpeg but saying `1.0` or `0.5` is less friendly than your typical `0-100` scale. Not sure what values we would have the `max` and `min` be but food for thought!

Reply:
The reason I have it this way is that there are no limits to the `ffmpeg` values. 0.75 would slightly lower volume, 2.0 would add some light saturation, 1000 would hyper distort it (think bass-boosted memes), -1.0 would reverse polarity, etc. This means you have all the flexibility of `ffmpeg`, and someone writing a bot could choose how to use the values, e.g. limit values from 0 to 100 and map them to 0 to 1.0.